torchani/extension/aev.cu

```diff
@@ -0,0 +1,584 @@
+#include "nvToolsExt.h"
+#include <chrono>
+#include <cub/cub.cuh>
+#include <cub/util_allocator.cuh>
+#include <cuda.h>
+#include <cuda_runtime.h>
+#include <iostream>

```

Comment:
Also this

Reply:
Removed all unused directives. Marking as resolved for all.